File: IP-Web/Models/UserTokens.cs
namespace IP_Web.Models;

public class UserTokens
{
    public string Token { get; set; } = string.Empty;
    public TimeSpan Validity { get; set; }
    public string RefreshToken { get; set; } = string.Empty;
    public Guid GuidId { get; set; }
    public string UserId { get; set; } = string.Empty;
    public string UserEmail { get; set; } = string.Empty;
    public DateTime ExpiredTime { get; set; }
    
}

File: IP-Frontend/src/api/models/user-tokens.ts
/* tslint:disable */
/* eslint-disable */
import { TimeSpan } from './time-span';
export interface UserTokens {
  expiredTime?: string;
  guidId?: string;
  refreshToken?: null | string;
  token?: null | string;
  userEmail?: null | string;
  userId?: null | string;
  validity?: TimeSpan;
}
